.github/workflows/phpstan.yml

```diff
@@ -4,11 +4,11 @@ on:
   # Run on all pushes and on all pull requests.
   # Prevent the build from running when there are only irrelevant changes.
   push:
-    paths-ignore:
-      - '**.md'
+    paths:
+      - '**.php'
```

Comment:
should we run phpstan also when we change phtml files?

Reply:
From its configuration files I can only see app/code/* directories included. Maybe someone else can confirm that.

Reply:
> should we run phpstan also when we change phtml files?

No. W/o proper docblocks phpstan can't even resolve $this for block classes.